python/cuopt/cuopt/linear_programming/data_model/data_model.py

```
@@ -194,6 +194,29 @@ def set_csr_constraint_matrix(self, A_values, A_indices, A_offsets):
+        ----------
+        Q_values : np.array dtype - float64
+            Values of the CSR representation of the quadratic_objective
+            matrix as a device floating point array.
+        Q_indices : np.array dtype - int32
+            Indices of the CSR representation of the quadratic_objective
+            matrix as a device integer array.
+        Q_offsets : np.array dtype - int32
+            Offsets of the CSR representation of the quadratic_objective
+            matrix as a device integer array.
+        """
+        super().set_csr_constraint_matrix(A_values, A_indices, A_offsets)
```

Flake8 (7.3.0)

[error] undefined name 'A_values'

(F821)